markets/bfp-market/storage.dump.sol

```diff
@@ -593,8 +593,8 @@ interface IOrderModule {
         uint256 limitPrice;
         uint256 keeperFeeBufferUsd;
         address[] hooks;
-        bool isOrderStale;
-        bool isOrderReady;
+        bool isStale;
+        bool isReady;
```

Comment:
This was just outdated before?

Reply:
Yeah I think 'The BFP Merge' was approved, merged, before a storage.dump.sol was updated 😞. It's `OrderDigest.{isStale,isReady}` in main.